src/screens/LoginScreen.tsx

```diff
@@ -0,0 +1,182 @@
+        contentContainerStyle={{ width: `300%` }}
+        showsHorizontalScrollIndicator={false}
+      >
+        {/* Placeholder Image Block */}
+        <Box
+          width={dimensions.width}
+          alignItems="center"
+          justifyContent="flex-start"
+        >
+          <Image
+            source={require('../assets/images/fireworks.png')}
+            style={styles.image}
+          />
+        </Box>
+
+        {/* Login Form Block */}
+        <Box
+          width={dimensions.width}
+          alignItems="center"
+          justifyContent="flex-start"
+          paddingTop={10}
+        >
+          <Box>
+            <Text fontWeight="bold" color='darkGrey' paddingBottom={2}>
+              Participant ID
+            </Text>
+            <TextInput
+              placeholder="Example: ANIXETY-jBSkjbckjb"
+              style={styles.loginField}
+              onChangeText={(text) => onChangeText(text)}
+              value={value}
+            />
+          </Box>
+
+          <Text variant="caption2" textAlign="center" paddingTop={10} paddingHorizontal={6}>
+            Please enter your Participant ID into the form above. You should of
+            recieved this in your experiment briefing.
+          </Text>
+        </Box>
+
+        {/* Spinner that stays on screen until experiment is loaded */}
+        <Box
+          width={dimensions.width}
+          alignItems="center"
+          justifyContent="center"
+          alignContent="center"
+          paddingTop={10}
+        >
+          <Spinner
+            isVisible
+            size={100}
+            type={'Bounce'}
+            color={palette.purple}
+          />
+        </Box>
+      </ScrollView>
+
+      {/* Push button to bottom of screen */}
+      <Box flex={1} justifyContent="flex-end" paddingBottom={6}>
+        {stage != Stages.Loading && (
+          <Button
+            backgroundColor="purple"
+            width={300}
+            height={60}
+            display="flex"
+            alignItems="center"
+            justifyContent="center"
+            borderRadius='m'
+            alignSelf="flex-end"
+            label={stage == Stages.Login ? 'Login' : 'Start Experiment'}
+            opacity={buttonDisabled ? 0.6 : 1}
+            disabled={buttonDisabled}
+            onPress={() => nextStage()}
+          />
+        )}
+      </Box>
+    </Box>
+  )
+}
+
+const styles = StyleSheet.create({
+  image: {
+    width: '100%',
+    maxWidth: 240,
+    height: 300,
+    resizeMode: 'contain',
+    opacity: 0.7,
+  },
+  loginField: {
+    height: 50,
+    width: 320,
+    backgroundColor: 'white',
+    borderColor: palette.purple,
+    borderWidth: 2,
+    borderRadius: 7,
+    paddingLeft: 10,
+  },
+  button: {
+    backgroundColor: palette.purple,
+    width: 300,
+    height: 60,
+    display: 'flex',
+    alignItems: 'center',
+    justifyContent: 'center',
+    borderRadius: 8,
+    alignSelf: 'flex-end',
+  },
```

Comment:
`.button` styles are not used.

Also just on button styling. If you find that this specific button style is used often, it's best to extract it out as a variant.

[Restyle docs on Variants](https://github.com/Shopify/restyle#variants).

The result is you would be able to just do `<Button variant="primary">` instead of specifying the colours and stuff.